search.py

```diff
@@ -26,6 +36,27 @@ def setQuery(self, searchIn):
         self.searchRaw = searchIn
     #end of setQuery
 
+    #set browser
+    def setBrowser(self, browser):
+        regex = {'chrome': '(google|chrome|google chrome|google-chrome)',
+                 'firefox': '(firefox|mozilla|mozilla firefox|mozilla-firefox)',
+                 'iexplore': '(ie|internet explorer|internet-explorer|iexplorer|iexplore)',
+                 'safari': 'safari'}
+        for i in regex:
+            match = re.search(regex[i], browser, re.IGNORECASE)
+            if match:
+                browser = i
+        self.browser = browser
+    #end of setBrowser
+
+    #get browser
+    def getBrowser(self):
+        try:
+            webbrowser.get(self.browser).open_new_tab(self.url)
+        except(webbrowser.Error):
+            printAvailableBrowsers(self.browser)
```

Comment:
I'm debating if we should open the search in the default browser if the selected browser is invalid. This could be resolved in another issue if necessary but this is something that's worth thinking about.